test/JDBC/input/1_GRANT_SCHEMA-vu-prepare.mix

```diff
@@ -161,4 +161,4 @@ go
 
 -- check for inconsistent metadata before upgrade
```

Comment:
Please add upgrade tests as well.

Reply:
Added. 
Future objects like Tables and views  created by the schema owner other than `dbo`  would throw permission error. But we have a simple workaround of granting the schema permission again, we can document this. Fixing this in upgrade involves iterating over all the schema permission entries in the catalog and then altering the default privilege of schema if the schema owner is not dbo/db_owner. Is it worth the effort?